### ISSUE TITLE
Feature request: Option to "Beautify"/"Uglify" when viewing the "structure" of a View.

### ISSUE THREAD

**Issue by daytonlowell** [TablePlus/TablePlus#1023]
I thought I may be able to use the "Format SQL" plugin as a workaround, but that threw an error because there was no "SQL Editor".

**Comment by huyphams** [comment]
it only formats the SQL inside the SQL Editor. you must open the SQL Editor. please check 
https://github.com/TablePlus/sql-formatter

**Comment by daytonlowell** [comment]
I guess what I'm really said for is that when you go to the "structure" screen on a View, it just opens a new SQL Query screen with the View statement already in the editor.

Rather than it being a special view. Only thing missing from the SQL Query screen that is useful for a view is the ability to toggle between Data/Structure for the table.

**Comment by jaimetobar** [comment]
I agree with this request - I end up copying the view structure into the SQL editor, do what I want and then copying/paste back.

**Comment by huyphams** [comment]
Hi if you have a chance, you could try this patch, it fixes all the issues:
https://www.dropbox.com/scl/fi/rnzl329j9llknuoj0c0h1/TablePlus.dmg?rlkey=fykl1eshbl1ylnn48s5ic7rau&st=00mt04zi&dl=0
